src/main/java/org/junit/Assume.java

```diff
@@ -79,6 +79,7 @@ public static void assumeFalse(String message, boolean b) {
      * If called with one or more null elements in <code>objects</code>, the test will halt and be ignored.
```

Comment:
Please update the Javadoc:

    If called with a {@code null} array or one or more {@code null} elements in {@code objects},
    the test will halt and be ignored.